exercises/atbash-cipher/AtbashCipher.example

```diff
@@ -0,0 +1,63 @@
+module AtbashCipher exposing (..)
+
+import String
+import Dict
+import Char
+import Regex exposing (HowMany(All), regex)
+
+
+encode : String -> String
+encode plain =
+  let
+    translate =
+      toTranslator alphabet tebahpla
+  in
+    plain
+      |> String.toLower
+      |> String.filter (\c -> Char.isLower c || Char.isDigit c)
+      |> String.map translate
+      |> insertEvery 5 " "
+
+
+decode : String -> String
+decode cipher =
+  let
+    translate =
+      toTranslator tebahpla alphabet
+  in
+    cipher
+      |> String.filter ((/=) ' ')
+      |> String.map translate
+
+
+alphabet : String
+alphabet =
+  "abcdefghijklmnopqrstuvwxyz"
+
+
+tebahpla : String
+tebahpla =
```

Comment:
i personally enjoy this variable name very much but it's more "the Elm way" to call it `reversedAlphabet` or something of that nature